src/elements/common/annotator-context/withAnnotations.tsx

```diff
@@ -89,6 +120,28 @@ export default function withAnnotations<P extends object>(
             annotator.emit('annotations_remove', id);
         };
 
+        createElementsError = (error: ElementsError | Error): ElementsError => {
+            const { message = '' } = error;
+            const origin = 'origin' in error && error.origin ? error.origin : ORIGIN_CONTENT_PREVIEW;
+
+            return {
+                code: ERROR_CODE_FETCH_ANNOTATIONS,
+                // eslint-disable-next-line @typescript-eslint/camelcase
+                context_info: error,
```

Comment:
I think the main purpose of `context_info` is to provide data that's not directly available in the error. Can you investigate how the feed API uses this?